packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -26,6 +26,33 @@ export const toggleGroupControl = ( {
 
 	${ toggleGroupControlSize( size ) }
 	${ ! isDeselectable && enclosingBorders( isBlock ) }
+
+
+	@media not ( prefers-reduced-motion ) {
+		&.is-animation-enabled::before {
+			transition-property: left, width;
```

Comment:
Could we switch to using `transform` instead, like we recently did with `Tabs`?

Reply:
That was the original implementation, but unfortunately it distorts border radius values. You can play with my animation sandbox here to see what I mean: https://stackblitz.com/edit/solidjs-templates-grt7kw?file=src%2FApp.tsx

Reply:
As a follow-up, we could look at https://css-tricks.com/various-methods-for-expanding-a-box-while-preserving-the-border-radius/#aa-changing-scale so that we could use `transform`

Reply:
Yeah, I'd like to look into it, I even had a chat with Matt Perry (Framer Motion author) who pointed me to the place in the codebase where these corrections happen. But yeah, let's better do it in a follow up. After the upcoming refactor, vertical tabs will potentially benefit from that technique too.